src/components/Tabs/Tabs.spec.jsx

```diff
@@ -1,101 +1,39 @@
-/* eslint-disable react/jsx-filename-extension */
-import { mount } from '@cypress/react18';
-import { Tabs } from './Tabs';
-
-const page = {
-  tabs: () => cy.byDataCy('Tab'),
-  content: () => cy.byDataCy('TabContent'),
-  tab: index => page.tabs().eq(index),
-  tabLink: index => page.tab(index).find('a'),
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const handleTabClick = tabId => {
+    if (tabId !== activeTabId) {
+      onTabSelected(tabId);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': tab.id === activeTab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                onClick={() => handleTabClick(tab.id)}
+                data-cy="TabLink"
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {activeTab.content}
+      </div>
+    </div>
+  );
 };
```

Comment:
This file seems to be a duplicate of the `Tabs.jsx` component. For a test specification file, you should write test cases to verify the component's behavior. Consider using a testing framework like Jest or React Testing Library to create tests for the `Tabs` component.